src/test/java/reposense/parser/ArgsParserTest.java

```diff
@@ -57,6 +60,15 @@ public void before() {
         DEFAULT_INPUT_BUILDER.reset().addConfig(CONFIG_FOLDER_ABSOLUTE);
     }
 
+    @After
+    public void after() {
+        try {
+            FileUtil.deleteDirectory(PROJECT_DIRECTORY.resolve(NONEXISTENT_DIRECTORY).toString());
```

Comment:
Can just do the deletion of directory in the test method that creates it?

Reply:
Actually, we cannot. The reason is that when the log is being used, there will be a `.lck` file that prevents the log file to be deleted.